x-pack/plugins/security_solution/server/endpoint/services/artifacts/manifest_manager/manifest_manager.ts

```diff
@@ -157,12 +157,7 @@ export class ManifestManager {
   }): Promise<WrappedTranslatedExceptionList> {
     if (!this.cachedExceptionsListsByOs.has(`${listId}-${os}`)) {
       let itemsByListId: ExceptionListItemSchema[] = [];
-      if (
-        (listId === ENDPOINT_ARTIFACT_LISTS.hostIsolationExceptions.id &&
-          this.productFeaturesService.isEnabled(ProductFeatureKey.endpointResponseActions)) ||
-        (listId !== ENDPOINT_ARTIFACT_LISTS.hostIsolationExceptions.id &&
-          this.productFeaturesService.isEnabled(ProductFeatureKey.endpointArtifactManagement))
-      ) {
+      if (this.productFeaturesService.isEnabled(ProductFeatureKey.endpointArtifactManagement)) {
```

Comment:
@dasansol92 Can you confirm that this change is needed and a valid one?

Reply:
Not, I don't think we should change this. If there are host isolation exceptions in place but there is a downgrade scenario, those shouldn't be taken into account when generating artifacts. That was the intention of that condition.

Reply:
Reverted

Reply:
Ok, but the `ProductFeatureKey`s that grant HIE privileges have changed. Shouldn't the condition now be:
```
if (
        (listId === ENDPOINT_ARTIFACT_LISTS.hostIsolationExceptions.id &&
          this.productFeaturesService.isEnabled(ProductFeatureKey.endpointHostIsolationExceptions)) ||
        (listId !== ENDPOINT_ARTIFACT_LISTS.hostIsolationExceptions.id &&
          this.productFeaturesService.isEnabled(ProductFeatureKey.endpointArtifactManagement))
      ) {
```

Reply:
It would also be a good idea to add a comment explaining this condition to avoid confusion in the future

Reply:
I think @semd is right here since the HIE feature has been moved outside of `endpointResponseActions`

Reply:
6e50a73b0337754c312730c8ff984a52109ecf86
Changed to what @semd proposed, I agree that this makes the most sense when the purpose of this check is now clear. Thanks!
cc @dasansol92